dis_snek/models/discord/channel.py

```diff
@@ -894,7 +894,7 @@ def permissions_for(self, instance: Snowflake_Type) -> Permissions:
 
     async def add_permission(
         self,
-        target: "PermissionOverwrite" | "models.Role" | "models.User" | "models.Member",
+        target: Union["PermissionOverwrite", "models.Role", "models.User", "models.Member"],
```

Comment:
Just gonna note that this technically _could_ be written as `target: "PermissionOverwrite | models.Role | models.User | models.Member"` as suggested on the Discord server, but I personally prefer `Union` here anyways.

Reply:
Yeah, I considered it, but IDE integration is generally better with Union if we're using quotes